src/server/middleware/quoteCart.ts

```diff
@@ -85,7 +85,13 @@ export const quoteCartSessionMiddleware: Router.IMiddleware<
       variables: { market: apiMarket, locale: isoLocale },
     })
 
-    const newQuoteCartId = result.data.data.onboardingQuoteCart_create.id
+    const newQuoteCartId = result.data.data?.onboardingQuoteCart_create.id
```

Comment:
Just a nit: `result.data` could be destructured on line 76 to `const { data } = ...` since we don't seem to use anything else from `result`